src/set_theory/cardinal.lean

```diff
@@ -525,6 +525,14 @@ quotient.eq.trans
 ⟨λ ⟨f⟩, ⟨equiv.ulift.symm.trans $ f.trans equiv.ulift⟩,
  λ ⟨f⟩, ⟨equiv.ulift.trans $ f.trans equiv.ulift.symm⟩⟩
 
+/-- A variant of `lift_mk_eq` with universes specialized as `w = max u v`.
+This is sometimes necessary to avoid universe unification issues. -/
+theorem lift_mk_eq' {α : Type u} {β : Type v} :
+  lift.{u v} (mk α) = lift.{v u} (mk β) ↔ nonempty (α ≃ β) :=
```

Comment:
is this `lift_mk_eq.{u v 0}`?

Reply:
Ah, I just see the docstring. I don't think that's quite right, because with `w = max u v` you get `  lift.{u (max u v)}` (which is equal to `lift.{u v}`).
I'm fine with adding a new definition, but could you at least prove this using `lift_mk_eq`?

Reply:
I'm pretty sure it's not possible to prove this from `lift_mk_eq`, because it is not a specialization. I've updated the doc string to reflect this.

With `w = max u v`, you get `lift.{u (max u v)}`, which unfortunately doesn't unify with `lift.{u v}`, even though it is of the same type:

```
example : lift.{u v} = lift.{u (max u v)} := rfl --- fails!
```

Reply:
I just checked, `lift_mk_eq.{u v 0}` is a valid proof of this (as I suggested before).

Reply:
Thanks! Sorry for not understanding at first.